modules/document_repository/README.md

```diff
@@ -0,0 +1,30 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a useful tool that provides studies with a centralized location for important documents. Files may be uploaded and organized under any user-defined category. Subcategories can also be defined by the user to be nested under parent categories.
```

Comment:
I think it should be up to the users to decide if it's useful, I'd take out the adjective. It can also be used for unimportant documents. Didn't we decide a while ago that the document repository should be for non-candidate documents? (Candidate documents go in the media module.)

The second sentence should be in the scope, not the purpose, and I'm not sure that the third is required.